services/prometheus/prometheus.go

```diff
@@ -101,7 +101,8 @@ func (svc *Service) Run(ctx context.Context) {
 			}
 
 			if err := svc.updateConfiguration(); err != nil {
-				svc.l.Errorf("Failed to update configuration: %+v.", err)
+				svc.l.Errorf("Failed to update configuration, will retry: %+v.", err)
+				svc.UpdateConfiguration()
```

Comment:
Why do we have two methods with almost same names?

Reply:
It's confusing

Reply:
Originally – for compatibility with another PR :) I will change it

Reply:
Done.